### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by marbx**
Improves the documentation:
- Corrects typos and increases readability
- Removes the dead link to the NSIS build procedure
- Removes an outdated, unofficial NSIS build procedure
- Describes the mockup

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -177,7 +179,8 @@ $pythonversion = 0
 if (Test-Path ..\salt-windows-nsis\build\Salt-Minion*Py3*.exe) {$pythonversion=3}
 if ($pythonversion -eq 0) {
   Write-Host -ForegroundColor Red "Cannot determine Python version from ..\salt-windows-nsis\build\Salt-Minion*.exe"
-  Write-Host -ForegroundColor Red "Have you build the NSIS Nullsoft exe installer?"
+  Write-Host -ForegroundColor Yellow "Have you build the NSIS Nullsoft exe installer?"
```
Comment (twangboy) [inline build.ps1:182]: built

File: build.ps1 [hunk 2]
```diff
@@ -132,7 +132,8 @@ if ($gitexe.length -eq 0) {
 
 if (-Not (Test-Path ..\salt)) {
   Write-Host -ForegroundColor Red "No directory ..\salt"
-  Write-Host -ForegroundColor Red "Have you build the NSIS Nullsoft exe installer?"
+  Write-Host -ForegroundColor Yellow "Have you build the NSIS Nullsoft exe installer?"
```
[the remark this answers is not in the document]
Comment (marbx) [reply build.ps1]: Fixed

File: build.ps1 (hunk 1 above)
Comment (marbx) [reply build.ps1]: Fixed

**Comment by marbx** [comment]
Hi Shane,
thanks for the review, please repeat or merge

good to see you again and David for the first time.
Markus

**Pull request merged**